risedev-profile-include.template.yml

```diff
@@ -0,0 +1,13 @@
+# This is a template file to show how to include extra profiles besides the ones defined in `risedev.yml`.
+#
+# To use this template:
+# - duplicate this file and rename it to `risedev-profile-include.<any>.yml`
```

Comment:
Why `.*.yml`? Isn't one user-defined fine enough? e.g., `.user.yml` 🤪

Reply:
You might be right. 🤡